operations/outletitems.md

```diff
@@ -41,120 +44,118 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
-            "Number": "1306",
-            "ClosedUtc": "2018-07-26T10:19:02Z",
-            "UpdatedUtc": "2018-07-26T10:19:02Z",
-            "Notes": null
-        }
-    ],
-    "Cursor": "8d02142f-31cf-4115-90bf-ae5200c7a1ba"
+{
+  "OutletItems": [
+    {
+      "Id": "f29821b7-1659-4c96-a8c7-3725d0f1509b",
+      "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "BillId": "5c82a9bd-729c-4f80-af48-a56ab3aebbf6",
+      "AccountingCategoryId": "1131ddd1-fa2b-4150-bbf6-7fce94941f65",
+      "Type": "Revenue",
+      "Name": "sample revenue item",
+      "UnitCount": 4,
+      "UnitAmount": {
+        "Currency": "EUR",
+        "GrossValue": 11,
+        "NetValue": 11,
+        "TaxValues": []
+      },
+      "CreatedUtc": "2018-07-25T12:47:11Z",
+      "ConsumedUtc": "2018-07-26T12:19:07Z",
+      "UpdatedUtc": "2018-07-26T12:19:07Z",
+      "ExternalIdentifier": null,
+      "Notes": null
+    },
+    {
+      "Id": "dfec07c6-e278-4ed0-932f-41bbd1f38039",
+      "EnterpriseId": "4d0201db-36f5-428b-8d11-4f0a65e960cc",
+      "BillId": "7bdd3b53-7bb3-419d-8ff2-c9bde65d0c7e",
+      "AccountingCategoryId": "7EDAB816-BF4E-40CC-8936-7BC0B222908D",
+      "Type": "Payment",
+      "Name": "sample payment item",
+      "UnitCount": 77,
+      "UnitAmount": {
+        "Currency": "EUR",
+        "GrossValue": 2,
+        "NetValue": 2,
+        "TaxValues": []
+      },
+      "CreatedUtc": "2018-07-25T16:25:28Z",
+      "ConsumedUtc": "2018-07-26T10:11:08Z",
+      "UpdatedUtc": "2018-07-26T10:11:08Z",
+      "ExternalIdentifier": "PaymentReference123",
+      "Notes": null
+    }
+  ],
+  "OutletBills": [
+    {
+      "Id": "5c82a9bd-729c-4f80-af48-a56ab3aebbf6",
+      "OutletId": "c9f09414-2fdf-41d6-bdb1-12158b01048e",
+      "Number": "1305",
+      "ClosedUtc": "2018-07-26T12:19:07Z",
+      "UpdatedUtc": "2018-07-26T12:19:07Z",
+      "Notes": null
+    },
+    {
+      "Id": "7bdd3b53-7bb3-419d-8ff2-c9bde65d0c7e",
+      "OutletId": "E0A29D6D-411E-4302-AA6D-9289935C5F14",
+      "Number": "1306",
+      "ClosedUtc": "2018-07-26T10:19:02Z",
+      "UpdatedUtc": "2018-07-26T10:19:02Z",
+      "Notes": null
+    }
+  ],
+  "Cursor": "8d02142f-31cf-4115-90bf-ae5200c7a1ba"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `OutletItems` | array of [Outlet item](#outlet-item) | required | The outlet items. |
-| `OutletBills` | array of [Outlet bill](#outlet-bill) | required | The outlet bills of the items. |
-| `Cursor` | string | required | Unique identifier of the last and hence oldest outlet item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older outlet items. |
+| `OutletItems` | array of [Outlet item](outletitems.md#outlet-item) | optional | The outlet items. |
+| `OutletBills` | array of [Outlet bill](outletitems.md#outlet-bill) | optional | The outlet bills of the items. |
```

Comment:
issue: These should be marked as required, because even if they're empty, we provide an empty array.